persistence/provider/sql/eventstore.go

```diff
@@ -64,20 +64,40 @@ type eventStoreDriver interface {
 		ak string,
 	) (uint64, error)
 
-	// SelectEvents selects events from the eventstore that match the given
-	// query.
+	// SelectEventsByType selects events from the eventstore that match the
+	// given event types query.
```

Comment:
```suggestion
	// given query.
```

Just changed this one back, since no behavior has changed there yet. This rename should probs have been a separate PR, but all good.

As usual, please duplicate these changes to the implementations in each provider.